module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.hpp

```diff
@@ -87,10 +87,13 @@ namespace module::behaviour::strategy {
         std::vector<message::behaviour::FieldTarget> lookTarget;
 
         // TODO: remove horrible
```

Comment:
at what point do we remove TODO's like this lol

Reply:
Probably now. Clean slate.